src/locale/locales/fr/messages.po

```diff
@@ -5030,6 +5049,10 @@ msgstr "Sélectionnez le(s) service(s) de modération destinataires du signaleme
 msgid "Select the service that hosts your data."
 msgstr "Sélectionnez le service qui héberge vos données."
 
+#: src/view/com/composer/videos/SelectVideoBtn.tsx:53
+msgid "Select video"
+msgstr "Sélectionne une vidéo"
```

Comment:
It's a label, so I would argue that it is, in fact, an infinitive we should use here.

```suggestion
msgstr "Sélectionner une vidéo"
```

Reply:
Oops, yes that makes sense 👍